ui/src/app/components/help-sidebar.tsx

```diff
@@ -582,6 +591,62 @@ export const HelpSidebar = fp.flow(
       </FlexRow>;
     }
 
+    withinPastTwentyFourHours(epoch) {
+      const completionTimeMoment = moment(epoch);
+      const twentyFourHoursAgo = moment().subtract(1, 'days');
+      return completionTimeMoment.isAfter(twentyFourHoursAgo);
+    }
+
+    displayExtractionIcon(icon: IconConfig) {
+      const {extractionJobs} = this.state;
+      const jobsByStatus = fp.groupBy('status', extractionJobs);
+      let status;
+      // If any jobs are currently running, show running icon.
+      if (jobsByStatus[TerraJobStatus.RUNNING]) {
```

Comment:
Started slack thread on the business logic for this - but your implementation of the logic as currently specified  looks good

Reply:
Looks like we have resolution there - do you want to update that in this PR?

Reply:
What do you mean - like leave a comment to that effect?

Reply:
As in: do you want to change the display logic to reflect what we discussed in the thread. Shimon's original suggested logic is still implemented here

Reply:
argh

rebase episode iii: revenge of the rebase